library/src/main/java/com/jmedeisis/draglinearlayout/DragLinearLayout.java

```diff
@@ -47,6 +47,42 @@
     private final float nominalDistanceScaled;
 
     /**
+     * Use with {@link com.jmedeisis.draglinearlayout.DragLinearLayout#setContainerScrollViewListener(com.jmedeisis.draglinearlayout.DragLinearLayout.ScrollViewListener)}
+     * to scroll a view which contains draggable view.
+     */
+    public interface ScrollViewListener {
```

Comment:
`Listener` sounds weird here. Listener sounds like something just listening for some events passively.
How about `Scrollable`?